fish-rust/src/builtins/ulimit.rs

```diff
@@ -0,0 +1,354 @@
+use std::cmp::max;
+
+use libc::{c_int, RLIM_INFINITY};
```

Comment:
Remove the second using here then add the following const definition:

```rust
const RLIM_INFINITY: u64 = libc::RLIM_INFINITY as u64;
```

That'll take care of most of the errors you have under 32-bit Ubuntu.